search-commons/src/test/resources/sample_publication_with_affiliations.json

```diff
@@ -176,5 +170,10 @@
     "owner": "kirTest@unit.no",
     "ownerAffiliation": "https://www.example.org/Bergen"
   },
-  "status": "PUBLISHED"
+  "status": "PUBLISHED",
+  "topLevelOrganization": [{
```

Comment:
Should we rename this field to topLevelOrganizations since it now contains multiple values?